test/integration/activate_int_test.go

```diff
@@ -423,6 +423,7 @@ version: %s
 `, constants.BranchName, constants.Version))
 
 	ts.PrepareActiveStateYAML(content)
+	ts.PrepareCommitIdFile("59404293-e5a9-4fd0-8843-77cd4761b5b5")
```

Comment:
Shouldn't the commit ID file be disabled for v43? Sorry if I'm missing something here...

Reply:
That function body will have a temporary shim to handle legacy vs. commit ID file. At this time since that PR is not merged, we're still using commit ID files.

Reply:
Does this change now that the PR is removed?

Reply:
The function ends up modifying the activestate.yaml to the `project: org/project#commitId` format instead of creating a commit file. Whenever we end up putting localcommits and buildscripts back, that function will go back to writing commit files instead of modifying activestate.yaml.